src/main/java/dgen/dataset/DatasetConfig.java

```diff
@@ -71,7 +77,10 @@ public static DatasetConfig specToConfig(DatabaseSpec databaseSpec) {
     }
 
     public static DatasetGenerator specToGenerator(DatabaseSpec databaseSpec) {
-        return new DatasetGenerator(specToConfig(databaseSpec));
+        long t0 = System.currentTimeMillis();
+        DatasetGenerator datasetGenerator = new DatasetGenerator(specToConfig(databaseSpec));
+        System.out.println("Created generators in " + (System.currentTimeMillis() - t0));
```

Comment:
If we want to keep these messages, then we should use a logger and have these messages as 'traces', with the normal log level configured to info. Either that or removing the performance/instrumentation code. I rather move to a logger and have this fine-grained information be part of the generation process. Let's chat tomorrow to brainstorm options.